### DISCUSSION TITLE
TFT35-E3 v3.0 updated to v27, now Unknown command: "M154 S0"

### DISCUSSION THREAD

**Discussion by frankmim** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1933] [Q&A]
Hello,
I've updated the firmware of my TFT35-E3 v3.0 today with the latest version 27 (from v26). Update went well and the printer is running fine except every few seconds a beep with the Unknown command: "M154 S0" error message. It keeps quiet during printing. Also got the skr mini E3 v1.2 installed.
Anybody an idea to fix this?

**Comment by kisslorand** [comment]
The only mode to resolve this (for the moment) is to compile the new Marlin bugfix version and enable `AUTO_REPORT_POSITION` in "Configuration_adv.h" (line 3422).

**Reply by frankmim** [reply]
Fixed, tnx!!!!
